demos/http/common/src/http_demo_url_utils.c

```diff
@@ -109,30 +140,54 @@ HTTPStatus_t getUrlAddress( const char * pUrl,
 
     if( httpStatus == HTTPSuccess )
     {
-        parserStatus = http_parser_parse_url( pUrl, urlLen, 0, &urlParser );
+        /* Search for the start of the hostname using the "://" separator. */
```

Comment:
Does the new parser not have functions for it?

Reply:
No it does not, it was removed in llhttp